server/listeners/eventHandlers/probationEvents/index.ts

```diff
@@ -1,12 +1,13 @@
 import { SQSMessage } from 'sqs-consumer'
 import logger from '../../../../logger'
 
-export default async function handleProbationEvents(messages: SQSMessage[]) {
-  messages.forEach(message => {
-    const deliusEvent = message.Body
-    logger.info(`Delius Event : ${deliusEvent}`)
+export default function buildEventHandler() {
+  return async (messages: SQSMessage[]) => {
+    messages.forEach(message => {
+      logger.info(`Delius Event : ${message.Body}`)
 
```

Comment:
Changes in preparation to accept required services?

Reply:
Yep, refactored it to be the same as the domain events one, so it can accept services as dependency